web/src/engine/websites/ScantradUnion.ts

```diff
@@ -0,0 +1,39 @@
+import { Tags } from '../Tags';
+import icon from './ScantradUnion.webp';
+import { DecoratableMangaScraper } from '../providers/MangaPlugin';
+import * as Common from './decorators/Common';
+
+function ChapterInfoExtractor(element: HTMLDivElement) {
+    const number = element.querySelector<HTMLSpanElement>('span.chapter-number').innerText.trim();
+    let title = element.querySelector<HTMLSpanElement>('span.chapter-name').innerText.trim();
+    title = title.length ? ' - ' + title : title;
+    title = (number + title).trim();
+    const id = element.querySelector<HTMLAnchorElement>('div.buttons a.btnlel:not([onclick])').pathname;
+    return { id, title };
+}
+
+const pageScript = `
+    let result = [];
```

Comment:
The `result` variable is assigned, but never used. May as well remove the variable.